packages/patternfly-4/react-table/src/components/Table/Table.d.ts

```diff
@@ -48,22 +48,27 @@ export interface IExtra extends IExtraData {
 }
 
 export interface ISortBy {
-  index?: Number;
+  index?: number;
   direction?: OneOf<typeof SortByDirection, keyof typeof SortByDirection>;
 }
 
 export interface IAction {
-  title: String;
+  title: string;
   onClick: (event: MouseEvent, rowIndex: number, rowData: IRowData, extraData: IExtraData) => void;
 }
 
 export interface ISeparator {
   isSeparator: Boolean
 }
 
+export interface ITransform {
+  isVisible: boolean;
+  children: React.ReactNode;
+}
+
 export interface ICell {
-  title: String;
-  transforms?: Array<Function>;
+  title: string;
+  transforms?: ((value: any) => ITransform)[];
   cellTransforms?: Array<Function>;
   columnTransforms?: Array<Function>;
   formatters?: Array<Function>;
```

Comment:
Nice way of providing further information o others trough typed array. We should add this to other arrays as well.
```TSX
  transforms?: ((value: any) => IDecorator)[];
  cellTransforms?: ((value: any) => IDecorator)[];
  columnTransforms?: ((value: any) => IDecorator)[];
  formatters?: ((value: any) => IDecorator)[];
```